cfme/fixtures/parallelizer/__init__.py

```diff
@@ -396,7 +398,26 @@ def pytest_runtestloop(self):
 
         """
         # Build master collection for slave diffing and distribution
-        self.collection = [item.nodeid for item in self.session.items]
+        for item in self.session.items:
+            is_serial = False
+            # check module
+            if hasattr(item.module, "pytestmark"):
```

Comment:
I'm not sure you need all this code at all. there should be appropriate api in py.test.
there is for example get_marker() method.

Reply:
i beleive it should be something like `item.get_closest_marker("serial")`

Reply:
Ah awesome! That's a whole lot nicer than what I have now :smile: